tests/bal-tools-integration-tests/src/test/resources/testng.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2025, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This header has to be LLC

Reply:
Fixed in [58e2f5b](https://github.com/ballerina-platform/ballerina-lang/pull/44104/commits/58e2f5bf5ac2e28035528205ab80ed59bbd5702f)